VideoLocker/src/main/java/org/edx/mobile/view/NavigationFragment.java

```diff
@@ -231,6 +233,7 @@ public void onClick(View v) {
             @Override
             public void onClick(View v) {
                 environment.getRouter().forceLogout(getActivity(), environment.getSegment(), environment.getNotificationDelegate());
+                SecurityUtil.clearUserData(MainApplication.application);
```

Comment:
Hmm, good point. It seems that my implementation for invalidating the refresh token from there in #800 was incorrect as well. Maybe there should be some other decoupled method for implementing manual logout that's invoked by the actual click listener on the logout button, which would then have the specific logic for manual logout, and then call through to this `forceLogout()` method.